python/tests/src/v5/test_access_token.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+import mock
+import json
+
+from src.v5.access_token import AccessToken
+from src.v5.oauth_scope import Scope
+
+class AccessTokenTest(unittest.TestCase):
+
+    @mock.patch('src.v5.access_token.requests.post')
+    @mock.patch('src.v5.access_token.get_auth_code')
+    def test_access_token(self, mock_get_auth_code, mock_requests_post):
+        mock_get_auth_code.return_value = 'test-auth-code'
+
+        mock_api_config = mock.Mock()
+        mock_api_config.app_id = 'test-app-id'
+        mock_api_config.app_secret = 'test-app-secret'
+        mock_api_config.api_uri = 'test-api-uri'
+        mock_api_config.redirect_uri = 'test-redirect-uri'
+        mock_api_config.oauth_token_dir = 'test-token-dir'
+        mock_api_config.version = 'v5'
+        mock_api_config.verbosity = 2
+
+        mock_response = mock.MagicMock()
+        mock_response.__str__.return_value = '<Response [200]>'
+        mock_response.json.return_value = {'status': 'test-status',
+                                           'scope': 'test-scope',
+                                           'access_token': 'test-access-token',
+                                           'refresh_token': 'test-refresh-token'
+                                           }
+        mock_requests_post.return_value = mock_response
+
+        access_token = AccessToken(mock_api_config)
+        access_token.oauth()
+
+        # mock does not figure out enum equality, so need to unpack arguments
+        # of get_auth_code in order to check the default value for scopes.
+        self.assertEqual(True, mock_get_auth_code.mock_calls[0][2]['refreshable'])
```

Comment:
```suggestion
        self.assertTrue(mock_get_auth_code.mock_calls[0][2]['refreshable'])
```

Reply:
Done. I verified that this is the only instance of an assertEqual that should be assertTrue or assertFalse.